### PR TITLE
chore: update vite range

### PR DESCRIPTION
**Pull request by patak-dev**
The plugin should be compatible with both Vite 3 and 4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "slash": "^3.0.0",
     "source-map": "^0.6.1",
     "unbuild": "^0.7.4",
-    "vite": "^3.0.0-beta.4",
+    "vite": "^3.0.0 || ^4.0.0",
```
Comment (ArnaudBarre) [inline package.json:58]: maybe just 4 for dev deps?

**Review by ArnaudBarre — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (patak-dev) [inline package.json:58]:
```suggestion
    "vite": "^4.0.0",
```

Comment (patak-dev) [reply package.json]: ya, better 👍🏼

**Comment by ArnaudBarre** [comment]
Missing lockfile 🥲

**Pull request merged**